client/src/components/RestaurantInfo.jsx

```diff
@@ -4,7 +4,15 @@ import RestaurantInfoEntry from './RestaurantInfoEntry.jsx';
 import MapContainer from './MapContainer.jsx';
 
 const RestaurantInfo = props => (
+<<<<<<< HEAD
+<<<<<<< HEAD
   <div id="restaurantInfo" className="container">
+=======
+  <div id="restaurantInfo" className="container hidden">
```

Comment:
Are you planning on removing this DIV in a later commit, or should there be two "container hidden" DIVs rendering?